automatic/flashplayeractivex/tools/chocolateyInstall.ps1

```diff
@@ -1,14 +1,13 @@
-﻿$packageName = '{{PackageName}}'
-$version = '{{PackageVersion}}'
+$packageName = 'flashplayeractivex'
+$version = '24.0.0.186'
+$majorVersion = '24'
 $installArgs = '/quiet /norestart REMOVE_PREVIOUS=YES'
```

Comment:
you don't need this variable, remove this and add it directly to the `$packageArgs.silentArgs` instead